docs/rfc/active/business_glossary/README.md

```diff
@@ -0,0 +1,224 @@
+| arrivalTime| TimestampTicks| N| | Time the price book was received by the TickCollector. 100s of Nanoseconds since 1st January 1970 (ticks)|
+| bid1Price| com.xxxx.yyy.schema.common.Price| N| **common.monetoryAmount**|The bid price with rank 1/29.|
+| bid1Size| int| N| market.bidSize| The amount the bid price with rank 5/29 is good for.|
+|--|--|--|--|--|--|--|
+| | | | | | | |
+
+### Stiching Together
+
+
+Business Glossary will be a first class entity where one can define the business terms and this will be similar to entities like Dataset, CorporateUser etc.
+Then extending the Dataset's aspect SchemaMetadata, that will be extending the SchemaField model to capture urn of the business term (optional).
+
+![high level design](business_glossary_rel.png)
+
+The above diagram illustrates how business glossaries will be connected to other entities. Business terms are modelled as BusinessTerm entities and example business terms are `Term-1`, `Term-2`, .. `Term-n`. In the above diagram from the dataset (`DS-1`) element `e11` is linked to business term `Term-2` and `e12` is linked to `Term-1`. At the same time from Dataset (`DS-2`) element `e24` linked the business term `Term-2`, `e22` with `Term-3` and `e24` with `Term-4`.
+
+## Metadata Model Enhancements 
+
+There will be 1 top level GMA [entities](../../../what/entity.md) in the design: businessTerm (Business Glossary).
+It's important to make businessTerm as a top level entity because it can exist without a Dataset and can be defined independently by the business team.
+
+### URN Representation
+We'll define a [URNs](../../../what/urn.md): `BusinessTermUrn`.
+These URNs should allow for unique identification of business term.  
+
+A business term  URN will look like below:
+```
+urn:li:businessTerm:(<<namespace>>,<<name>>)
+```
+
+### New Snapshot Object
+There will be new snapshot object will be definied to onboard business terms along with definitions
+
+```
+/**
+ * A metadata snapshot for a specific BusinessTerm entity.
+ */
+record BusinessTermSnapshot {
+
+  /**
+   * URN for the entity the metadata snapshot is associated with.
+   */
+  urn: BusinessTermUrn
+
+  /**
+   * The list of metadata aspects associated with the dataset. Depending on the use case, this can either be all, or a selection, of supported aspects.
+   */
+  aspects: array[BusinessTermAspect]
+}
+```
+### BusinessTermAspect
+There will be new aspect defined to capture the required attributes & ownership information
+
+```
+/**
+ * A union of all supported metadata aspects for a BusinessTerm
+ */
+typeref BusinessTermAspect = union[
+  BusinessTermProperties,
+  Ownership
+]
+```
+
+### Entity BusinessTermProperties
+
+```
+/**
+ * Properties associated with a BusinessTerm
+ */
+record BusinessTermProperties {
+
+  /**
+   * Name of business term
+   */
+  name: string
+
+  /**
+   * Definition of business term
+   */
+  name: string
```

Comment:
Going back to my previous comment. Since you are capturing these in this aspect, should we safely make the URN opaque. WDYT ?

Reply:
could you please clarify further, not sure whether I understand the same